lib/osf-components/addon/components/osf-navbar/auth-dropdown/template.hbs

```diff
@@ -11,7 +11,7 @@
                 <img src='{{this.user.profileImage}}&s=25' alt='{{t 'auth_dropdown.user_gravatar'}}'>
             </div>
             <div class='nav-profile-name'>{{this.user.fullName}}</div>
-            <span class='caret'></span>
```

Comment:
I'm glad this is now the "old way" of doing this 👀

Reply:
Agreed